app/src/main/AndroidManifest.xml

```diff
@@ -4,68 +4,38 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
 
-    <application android:networkSecurityConfig="@xml/network_security_config"
+    <application
         android:name=".util.ResourceUtil"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:networkSecurityConfig="@xml/network_security_config"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".activity.RegisterationSuccessActivity">
+        <activity android:name=".activity.LauncherActivity"
+            android:theme="@style/AppTheme.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
 
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
         </activity>
+        <activity android:name=".activity.RegisterationSuccessActivity"></activity>
```

Comment:
if it close single line then we can use `<activity android:name=".activity.RegisterationSuccessActivity"/>`